packages/DateRangePicker/test/DateRangePicker.cypress.js

```diff
@@ -0,0 +1,54 @@
+import moment from "moment";
+
+describe("<DateRangePicker />", () => {
```

Comment:
Curious about why these tests needed to by in Cypress, instead of Jest like the rest of them?

Reply:
Also, minor point, but the containing folder should probably be `tests` instead of `test`.

Reply:
It is hard to answer this question, because it is just like a feeling and not rational point. But I will try to explain.

1. It feels like there are a lot of visual things (not behavioral) (like floating elements). It can't be properly tested with Jest, because it requires real browser to render.
2. This tests are more like user scenarios. It feels that Cypress is more suitable for testing user scenarios.

But I have not strong opinion on it. So we can convert it to Jest if you think it is good idea.